dashboard/config/routes.rb

```diff
@@ -612,6 +612,7 @@
     namespace :lti do
       namespace :v1 do
         resource :feedback, controller: :feedback, only: %i[create show]
+        post '/sections/owners', to: 'lti_sections#bulk_update_owners'
```

Comment:
Please use the [PATCH Rest API method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH) for partial updating of a resource:
```suggestion
        resources :sections, only: [] do
          collection do
            patch :bulk_update_owners
          end
        end
```